packages/helm-charts/blockscout/templates/grafana-hook.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: "{{ .Release.Name }}"
+  annotations:
+    "helm.sh/hook": post-upgrade
+spec:
+  template:
+    spec:
+      serviceAccountName: {{ .Release.Name }}-rbac
+      initContainers:
+{{ include "celo.blockscout.initContainer.secrets-init" . | indent 10 }}
+      containers:
+      - name: post-install-grafana
+        image: eruizgar91/gcloud-jq:latest
+        imagePullPolicy: IfNotPresent
+        command:
+        - /tmp/pod/secrets-init
+        args:
+        - --provider
+        - google
+        - /bin/sh
+        - -c
+        - /opt/scripts/grafana-hook.sh
+        - |
+          trap "touch /tmp/pod/main-terminated" EXIT
```

Comment:
You don't need that, it's used for the `db-terminating-sidecar` sidecar container to detect when it terminates to implement a waiting mechanism for the cloudsql proxy container. It should be terminated when the main container has finished its work, otherwise, the second (cloudsqlproxy) container would work indefinitely. In your case you're not working with the db, so you can just run your script normally.